GitUtils.ps1

```diff
@@ -282,15 +281,30 @@ function Find-Pageant() {
 
 # Attempt to guess $program's location. For ssh-agent/ssh-add.
 function Find-Ssh($program = 'ssh-agent') {
+    $sshLocation = 'C:\Program Files\Git\usr\bin\'

```

Comment:
Is this actually necessary? I would expect the existing code in `Find-Ssh` to be able to resolve the correct location relative to the discovered `git` command.